README.md

```diff
@@ -1,2 +1,8 @@
 # hydrodyn
 1D Hydrodynamic Model for urban channels that I developed during my master's degree
+
+Developed by Anne Caroline Negrao
```

Comment:
I think there is not reason to avoid using tildes and accent marks, they are well supported by any system nowadays. So maybe use `Negrão` instead?